facilities.csv

```diff
@@ -1,4 +1,5 @@
 URI,name,street,zip,city,ticket_vending_machine,luggage_lockers,free_parking,taxi,bicycle_spots,blue-bike,bus,tram,metro,wheelchair_available,ramp,disabled_parking_spots,elevated_platform,escalator_up,escalator_down,elevator_platform,audio_induction_loop,sales_open_monday,sales_close_monday,sales_open_tuesday,sales_close_tuesday,sales_open_wednesday,sales_close_wednesday,sales_open_thursday,sales_close_thursday,sales_open_friday,sales_close_friday,sales_open_saturday,sales_close_saturday,sales_open_sunday,sales_close_sunday
+http://irail.be/stations/NMBS/008729560,Aulnoye aymeries,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Capital letter? Dash?
https://en.wikipedia.org/wiki/Aulnoye-Aymeries

Reply:
Will fix with a capital letter and a dash! https://media0.giphy.com/media/3o7aCRloybJlXpNjSU/giphy.gif?cid=3640f6095c98fc87416b5a2f4db13f66